terraform-provider/examples/provider/provider.tf

```diff
@@ -0,0 +1,13 @@
+terraform {
+  required_providers {
+    cosmo = {
+      source  = "terraform.local/wundergraph/cosmo"
+      version = "0.0.1"
+    }
+  }
+}
+
+provider "cosmo" {
+  cosmo_api_url = "cosmo_669b576aaadc10ee1ae81d9193425705"
```

Comment:
The provider is cosmo, I think we don't need to prefix the values with `cosmo_` again.

Reply:
It's like this for convenience as we use `COSMO_API_URL` in the environment variables. I am fine with both in general.

Reply:
How does the majority of providers handle it? I'd go with the simplest approach.

Reply:
```
provider "aws" {
  region     = "us-west-2"
  access_key = "my-access-key"
  secret_key = "my-secret-key"
}
```

```
provider "vultr" {
  api_key = "VULTR_API_KEY"
  rate_limit = 100
  retry_limit = 3
}
``` 

Not necessarily the majority, but let's go without the prefix even AWS ist not using something like aws_xxx (even though their environment variables do)

Reply:
Sounds good.